stdlib/database/jdbc/src/main/ballerina/src/java.jdbc/Module.md

```diff
@@ -0,0 +1,93 @@
+## Module overview
+
+This module provides the functionality required to access and manipulate data stored in any type of relational database 
+that is accessible via Java Database Connectivity (JDBC). 
+
+**Prerequisite:** Please add the JDBC driver corresponding to the database you are trying to interact with, 
```

Comment:
```suggestion
**Prerequisite:** Add the JDBC driver corresponding to the database you are trying to interact with
```
The best practice is to avoid using "please" in technical documentation.